video/out/vo_rpi.c

```diff
@@ -720,11 +721,28 @@ static struct mp_image *take_screenshot(struct vo *vo)
     return NULL;
 }
 
+static void set_fullscreen(struct vo *vo) {
+    struct priv *p = vo->priv;
+
+    if (p->renderer_enabled)
+	set_geometry(vo);
+    vo->want_redraw = true;
+}
+
 static int control(struct vo *vo, uint32_t request, void *data)
 {
     struct priv *p = vo->priv;
 
     switch (request) {
+    case VOCTRL_VO_OPTS_CHANGED: {
+        void *opt;
+        while (m_config_cache_get_next_changed(p->opts_cache, &opt)) {
+            struct mp_vo_opts *opts = p->opts_cache->opts;
+            if (&opts->fullscreen == opt)
+                set_fullscreen(vo);
+        }
+        return 1;
```

Comment:
Nit but for consistency's sake this should return `VO_TRUE`.

Reply:
Noted. I based my code on b6fa63b01e, so should probably change that too.

Reply:
Hmm you're right, that VO mixes 1's, true's, and VO_TRUE (doesn't really matter but still).